docs/catchup.md

```diff
@@ -0,0 +1,33 @@
+# Ledger catchup
+
+A node uses a process called `catchup` to sync its ledgers with other nodes. It does this process after starting up 
```

Comment:
Please use bullet points to mention all cases where we can start catch-up more explicitly.

Reply:
Done